.github/workflows/check.yml

```diff
@@ -17,44 +17,52 @@ jobs:
       run:
         shell: bash
     strategy:
+      fail-fast: true
```

Comment:
Implicitly true?

Reply:
was a holdover from setting it false for testing, nbd to leave it imo

Reply:
🤷🏼‍♂️